File: backend/src/main/java/com/godeltech/mastery/backend/domain/dto/responce/MaintenanceDTO.java
package com.godeltech.mastery.backend.domain.dto.responce;

import java.util.List;

public record MaintenanceDTO(
  Long id,
  String operationNumber,
  Integer mileage,
  Integer timeInterval,
  List<MaintenanceOperationDTO> operations,
  List<DetailDTO> details
) {}

File: backend/src/main/kotlin/com/godeltech/mastery/backend/domain/dto/responce/MaintenanceDto.kt
package com.godeltech.mastery.backend.domain.dto.responce

data class MaintenanceDto(
    val id: Long?,
    val operationNumber: String?,
    val mileage: Int?,
    val timeInterval: Int?,
    val operations: List<MaintenanceOperationDto>?,
    val details: List<DetailDto>?
)
